aries-core/src/js/components/core/Selector/Selector.js

```diff
@@ -10,8 +11,10 @@ import { SelectorHeader } from './SelectorHeader';
 const StyledBox = styled(Box)`
   ${props =>
     props.selected &&
-    // eslint-disable-next-line max-len
-    `box-shadow: inset 0 0 0 1px ${props.theme.global.colors['border-selected']};`}
+    `box-shadow: inset 0 0 0 ${
+      props.theme.global.edgeSize['5xsmall']
```

Comment:
I think I misled you with edgeSize. Don't we have [border size](https://github.com/grommet/grommet-theme-hpe/blob/1205e72a8c493ab365b817722e8d5f681c2c848e/src/js/themes/hpe.js#L127)?

Also, shouldn't these dimensions be consuming design tokens instead of hard coded? If so, can we log an issue?

```suggestion
      props.theme.global.borderSize['small']
```

Reply:
Oops yes we should use borderSize, just pushed up that change. Here is an updated screenshot:

<img width="1223" height="419" alt="Screenshot 2025-10-30 at 3 43 14 PM" src="https://github.com/user-attachments/assets/8ea222b3-629e-43d0-8fab-1e8f4ba63731" />

> Also, shouldn't these dimensions be consuming design tokens instead of hard coded? If so, can we log an issue?

Yes, I'll file a ticket for this

Reply:
Ticket created https://github.com/grommet/grommet-theme-hpe/issues/558